src/frameworks/TYPO3.md

```diff
@@ -0,0 +1,17 @@
+# Getting started
+
+## Prerequisites
+
+**Composer**
+
+Composer is a tool for dependency management in PHP. It allows you to
+declare the dependent libraries your project needs and it will install
+them in your project for you.
```

Comment:
Any new content should use soft-wrapping only.  We haven't finished cleaning up old sections yet because Meh.  But new pages should all be soft-wrapped, not hard wrapped.